wiki/Tournaments/TAG Tournament/T2T/2019/en.md

```diff
@@ -0,0 +1,198 @@
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 2 months of osu!supporter (per player) |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month of osu!supporter (per player) |
+
+--------------------------------
+
+## Staff
+
+The TAG2 Tournament 2019 is run by various community members by distributing the multitude of tasks into various fields of responsibility.
+
+| Position | Member(s) |
+| --: | :-- |
+| Hosts | ![][flag_FI] [Mara](https://osu.ppy.sh/users/194294), ![][flag_GB] [Malteser](https://osu.ppy.sh/users/5218178) |
+| Co-Hosts | ![][flag_FI] [Lefafel](https://osu.ppy.sh/users/2295850), ![][flag_GB] [Malteser](https://osu.ppy.sh/users/7068973) |
+| Mappoolers | ![][flag_CA] [Legless](https://osu.ppy.sh/users/3224243), ![][flag_GB] [Doughy](https://osu.ppy.sh/users/5275937), ![][flag_FI] [Mara](https://osu.ppy.sh/users/194294), ![][flag_GB] [Malteser](https://osu.ppy.sh/users/5218178)|
+| Referees | ![][flag_GB] [Malteser](https://osu.ppy.sh/users/7068973), ![][flag_GB] [Malteser](https://osu.ppy.sh/users/5218178), ![][flag_FI] [Lefafel](https://osu.ppy.sh/users/2295850), ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_BR] [Flow-](https://osu.ppy.sh/users/4222824), ![][flag_PL] [HAEN24](https://osu.ppy.sh/users/4390077), ![][flag_VN] [steve_04_](https://osu.ppy.sh/users/10852911), ![][flag_FI] [Laurakko](https://osu.ppy.sh/users/7253731), ![][flag_MY] [Stupid Idiot](https://osu.ppy.sh/users/8355574), ![][flag_US] [YoshiLover456](https://osu.ppy.sh/users/6843383), ![][flag_GR] [nikolomara](https://osu.ppy.sh/users/10077264) |
+| Spreadsheets | ![][flag_TR] [oralekin](https://osu.ppy.sh/users/7631823), ![][flag_NO] [YokesPai](https://osu.ppy.sh/users/6399568) |
+| Graphics | ![][flag_GB] [Doomsday is Bad](https://osu.ppy.sh/users/3481378), ![][flag_FI] [Mara](https://osu.ppy.sh/users/194294) |
+
+----------------------------------
+
+## Tournament Schedule
+
+| Event | Date |
+| --: | :-- |
+| Registration Phase | 2019-02-15 / 2019-03-15 |
+| Qualifiers | 2019-03-30 / 2019-03-31 |
+| Group Stage | 2019-04-06 / 2019-04-07 |
+| Round of 16 | 2019-04-13 / 2019-04-14 |
+| Quarterfinals | 2019-04-20 / 2019-04-21 |
+| Semifinals | 2019-04-27 / 2019-04-28 |
+| Finals | 2019-05-04 / 2019-05-05 |
+| Grand Finals | 2019-05-11 / 2019-05-12 |
+
+--------------------------------
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/869513)
+- [Registration form](https://docs.google.com/forms/d/e/1FAIpQLSc412ThxxCb10gzYy7v6uHox4vffuozopRs8NzwxUx8uVoUVQ/viewform?usp=sf_link)
+- [TAG Club Discord](https://discord.gg/VKt5kGa)
+- [TAG Club Twitter](https://twitter.com/TAGClub_osu)
+- [Challonge](https://challonge.com/T2T_2019)
+- **[Spreadsheet](https://docs.google.com/spreadsheets/d/13f9zVN8n1NjBqwK7C23W5iOW-7Y2d9np9Di1WnlUfjg/edit#gid=1150595067)**
+
+
+--------------------------------
+
+## Groups
```

Comment:
can delete all of these sections up through Mappools, and add them back with later updates to this article